collective_effects_installation_tutorial.txt

```diff
@@ -10,8 +10,8 @@
 
 2) The second step is to create an SSH key to clone the git repositories:
 
-- cd .ssh
 - ssh-keygen

```

Comment:
I think you should add a condition here. In case the key already exists, it is not necessary (not desirable, actually) to create it again.